atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -77,7 +77,7 @@ def next_release_general(
         """Get next release for build."""
         if is_scratch_build(self.workflow):
             # no need to append for scratch build
-            next_release = self.workflow.user_params.get('pipeline_run_name')
+            next_release = self.workflow.user_params['pipeline_run_name']
```

Comment:
Why this change? to make it fail with `KeyError` when `pipeline_run_name` is not available?

Reply:
IIUC: the code doesn't work if `next_release` is None, this change ensures that it's a string (or KeyError, yeah)